src/components/CreateVmWizard/CreateVmWizard.js

```diff
@@ -95,12 +95,18 @@ function getInitialState (clusters, templates, blankTemplateId, operatingSystems
     const memoryInB = blankTemplate.get('memory')
     const cpus = blankTemplate.getIn([ 'cpu', 'vCPUs' ])
     const initEnabled = blankTemplate.getIn([ 'cloudInit', 'enabled' ])
+    const topology = blankTemplate.getIn([ 'cpu', 'topology' ])
 
     blankTemplateValues = {
       operatingSystemId,
       memory: memoryInB / (1024 ** 2), // bytes to MiB
       cpus,
       initEnabled,
+      topology: {
+        cores: parseInt(topology.get('cores'), 10),
+        sockets: parseInt(topology.get('sockets'), 10),
+        threads: parseInt(topology.get('threads'), 10),
```

Comment:
I don't think the `parseInt()` is necessary on these fields.  If for some reason they are strings, it makes more sense to cast them to a number in the transform function:

https://github.com/oVirt/ovirt-web-ui/blob/2ad01a8885b1f07aa32a455eb1be10a9f62bce5a/src/ovirtapi/transform.js#L378-L385